tests/e2e.sh

```diff
@@ -6,10 +6,12 @@
 # installs all the apps under testdata and checks that they pass or fail, depending on their
 # success condition.
 
-cd $(dirname $0)
+TESTDIR="$(pwd)/tests"
```

Comment:
One thing with `make` is that you can run from a different directory via `make -C draft test-e2e`, which means assuming that we are running from $GOPATH/src/github.com/Azure/draft is not always going to be the case. `cd $(dirname $0)` should be the right way to set this up; $0 is the full path to the script ($GOPATH/src/github.com/Azure/draft/tests/e2e.sh) so `cd $(dirname $0)` *should* put us in the right directory. Do you have a specific error message that can help illuminate the issue?

Reply:
Following the docs at https://github.com/Azure/draft/blob/master/docs/contributing/hacking.md#test-your-changes, I was running `make test-e2e` which would error trying to source the test helpers after it had cd'ed:

```
$ make test-e2e
./tests/e2e.sh
./tests/e2e.sh: line 12: ./tests/tools: No such file or directory
```

Reply:
Ah okay! So maybe we can then just change line 12 to `. ./tools` and that should fix the whole problem. How does that sound?

Reply:
Sounds good, running `make test-e2e` works with this:

```
cd $(dirname $0)
PATH="$(pwd)/../bin:$PATH"

. ./testhelpers.sh

```